applications/app/views/fragments/galleryBody.scala.html

```diff
@@ -66,8 +66,8 @@ <h2 class="fc-container__header__title">
                     itemprop="contentUrl" />
 
                 <span class="rounded-icon gallery2__fullscreen">
-                    <i class="i i-expand-white"></i>
-                    <i class="i i-expand-black"></i>
+                    @fragments.inlineSvg("expand-white", "icon", List("i"))
```

Comment:
also, is this just a hover state that changes the colour? if so, we should be able to get away with just one svg there, and set the `fill` using `:hover`